GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -196,6 +196,15 @@ GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage() {
       resultData = [NSKeyedArchiver archivedDataWithRootObject:obj];
 #pragma clang diagnostic pop
       if (archivePath) {
+        BOOL result = [[NSFileManager defaultManager]
```

Comment:
The lines 173-188 look mostly identical to 198-213. I wonder, can we avoid duplication somehow?

Reply:
Done.